client/src/models/Person.js

```diff
@@ -412,6 +412,15 @@ export default class Person extends Model {
       : Person.advisorShowPageOrderedFields
   }
 
+  /**
+   * @returns Keys of fields which should span over 2 columns
+   */
+  getFullWidthFields() {
+    return this.isPrincipal()
+      ? Settings.fields.principal.person.showAsFullWidthFields
+      : Settings.fields.advisor.person.showAsFullWidthFields
```

Comment:
Perhaps safeguard this as well with an ` || []` at the end?

Reply:
Ah crap, added that to the wrong code segment